lms/static/js/student_account/views/account_settings_factory.js

```diff
@@ -124,10 +124,11 @@
                             view: new AccountSettingsFieldViews.DropdownFieldView({
                                 model: userAccountModel,
                                 required: true,
-                                title: gettext('Country or Region'),
+                                title: gettext('Country of Full-Time Residence'),
                                 valueAttribute: 'country',
                                 options: fieldsData.country.options,
-                                persistChanges: true
+                                persistChanges: true,
+                                helpMessage: gettext('This field represents the country of full-time residence not the country of origin') // eslint-disable-line max-len
```

Comment:
Thanks @Ayub-Khan! Here is my suggestion:
"The country or region where you live."
(@schenedx I decided against including "full-time" - it might actually create more confusion, for someone trying to understand the definition of full-time". Do you agree?)